test/easyconfigs/easyconfigs.py

```diff
@@ -493,6 +493,8 @@ def check_dep_vars(self, gen, dep, dep_vars):
             'Boost.Python': [('1.64.0;', [r'EMAN2-2\.3-'])],
             # GATE 9.2 requires CHLEP 2.4.5.1 and Geant4 11.0.x
             'CLHEP': [('2.4.5.1;', [r'GATE-9\.2-foss-2021b'])],
+            # ParaView 5.10.x requires FFmpeg 4.x
+            'FFmpeg': [(r'4\.4\.', [r'ParaView-5\.10\.', 'OpenFOAM-v2206'])],
```

Comment:
@casparvl this can be removed now?

Reply:
@casparvl I accidentally deleted `test/easyconfigs/easyconfigs.py` (I just wanted to remove your changes)
I've made a PR to your branch to add it back, see https://github.com/sara-nl/easybuild-easyconfigs/pull/6

Reply:
Yes, it can be removed. As far as I could see, the `easyconfigs.py` that you now put back was the one in which the changes were already removed, right? I.e. I don't need to do this myself anymore?

Reply:
yes, thanks!